Modules/ITS/src/ITSClusterTask.cxx

```diff
@@ -239,23 +248,34 @@ void ITSClusterTask::monitorData(o2::framework::ProcessingContext& ctx)
           hGroupedClusterSizeLayerSummary[lay]->Fill(npix);
         }
       }
+
+      // Transformation to the local --> global
+      mGeom->fillMatrixCache(o2::math_utils::bit2Mask(o2::math_utils::TransformType::L2G));
```

Comment:
Is it necessary to update matrix for each cluster?

Reply:
You're right. I have moved it at the beginning